### ISSUE TITLE
Length of varchar column remains unchanged after migration

### ISSUE THREAD

**Issue by yorndv** [HackSoftware/django-enum-choices#36]
```
class StatusBEnum(Enum):
    WAITING = 'waiting'
    READY = 'ready'

status_B = EnumChoiceField(StatusBEnum, default=StatusBEnum.WAITING)
```
-> 
```
python manage.py makemigrations - OK
python manage.py migrate - OK
status_B varchar(7) - OK
```
-> 
```
class StatusBEnum(Enum):
    WAITING = 'waiting'
    CANCELLED = 'cancelled'
    READY = 'ready'

status_B = EnumChoiceField(StatusBEnum, default=StatusBEnum.WAITING)
```
-> 
```
python manage.py makemigrations - OK
python manage.py migrate - OK
status_B varchar(7) - WRONG - exptected varchar(9)
```

**Comment by RadoRado** [comment]
@yorndv thanks for opening the issue :+1: Will take a look

**Comment by RadoRado** [comment]
This is fixed by #37 & will be released in `2.1.1`
